broker-cli/commands/order/trade-history.js

```diff
@@ -0,0 +1,25 @@
+const BrokerDaemonClient = require('../../broker-daemon-client')
+const { handleError } = require('../../utils')
+
+/**
+ * sparkswap order trade-history
+ *
+ * ex: `sparkswap order trade-history'
+ *
+ * @param {Object} args
+ * @param {Object} opts
+ * @param {string} opts.rpcaddress
+ * @param {Logger} logger
+ */
+async function tradeHistory (args, opts, logger) {
+  const { rpcAddress } = opts
+  try {
+    const client = new BrokerDaemonClient(rpcAddress)
+    const tradeHistory = await client.orderService.getTradeHistory({})
+    logger.info(JSON.stringify(tradeHistory, null, 2))
```

Comment:
I assume there is a ticket for turning this into a table?

Reply:
Just added one. I felt like this PR was already huge but ill do it once this is merged.

Reply:
@martineehrlich it actually might be something to punt for later. It's a nice to have.